File: app/src/main/java/com/vm/talk2me/network/NetworkState.java
package com.vm.talk2me.network;

import android.content.Context;
import android.net.ConnectivityManager;

/**
 * Created by Van Manh on 23-thg 8-17.
 */

public class NetworkState {

    public static boolean isNetworkAvailable(Context context) {
        ConnectivityManager manager = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
        if (manager.getActiveNetworkInfo() != null && manager.getActiveNetworkInfo().isConnectedOrConnecting()) {
            return true;
        } else {
            return false;
        }
    }
}

File: app/src/main/java/com/vm/talk2me/network/NetworkState.kt
package com.vm.talk2me.network

import android.content.Context
import android.net.ConnectivityManager

/**
 * Created by Van Manh on 23-thg 8-17.
 */

object NetworkState {

    fun isNetworkAvailable(context: Context): Boolean {
        val manager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
        return if (manager.activeNetworkInfo != null && manager.activeNetworkInfo.isConnectedOrConnecting) {
            true
        } else {
            false
        }
    }
}
